cpu.cpp

```diff
@@ -31,7 +31,12 @@ void Cpu::initializeRegisters()
     memset(R, 0, sizeof(R));
 }
 
-void Cpu::setMemory(u8 *data, size_t size) {
+std::vector<u8> Cpu::getMemory() {
+    return memory;
+}
+
+void Cpu::setMachineCodeInMemory(u8 *data, size_t size) {
+    memory.erase(memory.begin(), memory.begin() + size);
```

Comment:
What meaning has size variable in this context? If it's total memory size, then ok.
But if it's the size of memory to be written, then erasing just memory.begin() + size it's not ok. What happens if we write a smaller memory than the previous one?

Reply:
Will fix